dashboard/app/models/levels/weblab.rb

```diff
@@ -55,10 +55,15 @@ def non_blockly_puzzle_level_options
       level_prop = {}
 
       properties.keys.each do |dashboard|
```

Comment:
This is localizing all of the properties. Within the blockly file, only specific properties get localized. Thoughts on which way we should go?